src/appleseed/renderer/modeling/project/assethandler.cpp

```diff
@@ -208,6 +208,13 @@ bool AssetHandler::handle_asset(string& asset_path) const
     APIString search_path;
     m_project.search_paths().qualify(asset_path, &qualified_asset_path, &search_path);
 
+    // Relative asset path, that does not exist. Handling is skipped.
+    // Means that this file is in unprovided search path (therefore should be provided by parent project)
+    if (search_path.empty() && !exists(qualified_asset_path.c_str()))
```

Comment:
Not really: you know that the asset path is relative because the case where it is absolute is handled at the beginning of the method.

Checking `search_path.empty()` means checking that there are no search paths at all for this project.